include/sentry.h

```diff
@@ -24,83 +24,155 @@ extern "C" {
 #endif
 #endif
 
+/*
```

Comment:
If you use `/**` here, we could use doxygen to generate code docs.

Reply:
That's good to know! Let's do that together with adding doxygen